doc/specific_iocs/plcs/OPCUA.md

```diff
@@ -0,0 +1,110 @@
+# OPC UA 
+## What is OPC UA
+OPC UA is a cross-platform, open-source, IEC62541 standard for data exchange from sensors to cloud applications developed by the OPC Foundation. It is characterised by: 
+
+* Standardized data models freely available for over 60 types of industrial equipment, published by the OPC Foundation via Companion Specifications
+* Extensible security profiles, including authentication, authorization, encryption and checksums
+* Extensible security key management, including X.509, token and password
+* Support for both client-server and publish-subscribe communication patterns
+* Communication protocol independent. Mappings to several communication protocols like TCP/IP, UDP/ 
+IP, WebSockets, AMQP and MQTT are specified
+* Initially successful in standardized data exchange with industrial equipment (discrete manufacturing, process manufacturing, energy) and systems for data collection and control, but now also leveraged in building automation, weighing and kitchen equipment and cloud applications
+* [Open](https://en.wikipedia.org/wiki/Open_standard) – open-source reference implementations freely available to OPC Foundation members, non members under GPL 2.0 license[2]
+* [Cross-platform](https://en.wikipedia.org/wiki/Cross-platform) – not tied to one operating system or programming language
+* [Service-oriented architecture](https://en.wikipedia.org/wiki/Service-oriented_architecture) (SOA)
+* The specification is freely available on the OPC Foundation website and is split into several parts 
+to ease implementation, but only OPC UA stack vendors need to read them, end users simply leverage 
+existing commercial and/or open-source stacks available in all popular programming languages
+
+More info can be found [here on Wikipedia for a general overview](https://en.wikipedia.org/wiki/OPC_Unified_Architecture), or for a more detailed description from [OPC Foundation](https://opcfoundation.org/about/what-is-opc/).
+
+## How is authentication handled?
```

Comment:
I get that you probably copied these headers from a message/comment I sent previously - but I think they might be clearer on the wiki as headings like "Authentication", "Communication", "Troubleshooting" etc.

I've put this comment once but applies to most of the headers in this file